api/api.go

```diff
@@ -51,15 +52,28 @@ func New(db *sqlx.DB, token string) *echo.Echo {
 			"Flakes": flakes,
 		})
 	})
-	app.GET("/ui/test/:name/fails", func(c echo.Context) error {
-		name := c.Param("name")
-		fails, err := api.failedBuilds(name)
+	app.GET("/ui/test/:id/fails", func(c echo.Context) error {
+		testID := c.Param("id")
+		fails, err := api.failedBuilds(testID)
```

Comment:
fails looks like a weird name for this, maybe testFailure, or just test? test.ID, test.Name, test.Builds

Reply:
True, renamed to test.